docs/trovo.md

```diff
@@ -0,0 +1,20 @@
+# Integrating the Trovo Provider
```

Comment:
We can keep this anyway, but we don't generally bother adding documentation for providers that don't have any custom settings.